.changeset/healthy-carpets-scream.md

```diff
@@ -0,0 +1,5 @@
+---
+"@nomicfoundation/slang": patch
+---
+
+Accept unicode in string literals in 0.4.11
```

Comment:
I think this is already covered by `.changeset/eighty-moons-talk.md`?

Reply:
You're right, we can keep this out, then.